matchmaking/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM eclipse-temurin:17-jre-alpine
+ARG JAR
+ARG LIB
+
+ARG APP_USER=agent
+ARG APP_UID=10001
+ARG APP_GID=30000
+
+RUN addgroup --gid "$APP_GID" --system "$APP_USER"
+
+RUN apk add --update curl 
```

Comment:
Based on this [TRG](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02#use-base-image-as-is) it is discouraged to modify a base image like this. It would make the image scan and package list provided by DockerHub unusable.

Reply:
will be resolved. Thx for the heads up. Martin used curl instead of wget for testing on his machine and commited the change unfortunately.